rpcs3/Emu/RSX/GCM.cpp

```diff
@@ -762,6 +762,34 @@ rsx::primitive_type rsx::to_primitive_type(u8 in)
 	throw new EXCEPTION("Unknow primitive type %d", in);
 }
 
+enum
+{
+	CELL_GCM_WINDOW_ORIGIN_TOP = 0,
```

Comment:
I prefer that the non class enum are not available outside of GCM.cpp so that all code uses the enum class ones.